### PR TITLE
Make the entire card clickable on Installed Charts view

### LINKED ISSUE #81: Make items on Installed charts page to be correct hyperlinks
To be able to open releases in a new browser tab, intuitively.

Installed charts in list should be correct links, and full row should be a link, not just header text.

Top menu items Installed/Repository should be correct links.

Revision links on the left should be correct links.

### PR DESCRIPTION
**Pull request by hdm23061993**
fixes #81 

## Enhancements

To be able to open releases in a new browser tab, intuitively.

✅ Installed charts in list should be correct links, and full row should be a link, not just header text.

Top menu items Installed/Repository should be correct links.

Revision links on the left should be correct links.

✅ Rev list full row should be a link

### CODE REVIEW

**Comment by undera** [comment]
We've had it this way in the past, and the experience were confusing when you wanted to select some text in that row, to copy-paste it. It were causing the UI to react to a click. 

The solution would have to consider that and do not react if the text is selected.

**Comment by hdm23061993** [comment]
Handled as per your comments.
Please see if this is what was expected.

File: pkg/dashboard/static/revisions-view.js [hunk 1]
```diff
@@ -32,7 +32,7 @@ function fillChartHistory(data, namespace, name) {
             $("#specRev").val(elm.revision).data("last-rev", elm.revision).data("last-chart-ver", elm.chart_ver)
         }
 
-        const rev = $(`<li class="px-2 pt-5 pb-4 mb-2 rounded border border-secondary bg-secondary position-relative">
+        const rev = $(`<li class="px-2 pt-5 pb-4 mb-2 rounded border border-secondary bg-secondary position-relative link">
```
Comment (undera) [inline pkg/dashboard/static/revisions-view.js:35]: If I select text in rev row - will it cause reaction of click?

Comment (hdm23061993) [reply pkg/dashboard/static/revisions-view.js:35]: nope, it just selects the text
Can you try it on your local and see if the change meets the expectation?

**Comment by hdm23061993** [comment]
@undera did you manage to review?

**Comment by undera** [comment]
I tried it. And just as I wrote in the comments: the attempt to select text in revision list leads to handling of click, which is not intuitively expected. 

I would make the same check in the revision click handler, to not do anything if there is selection happened

**Comment by hdm23061993** [comment]
Let me look into it

File: pkg/dashboard/static/revisions-view.js (hunk 1 above)
Comment (hdm23061993) [reply pkg/dashboard/static/revisions-view.js:35]: Fixed in next patch

**Comment by hdm23061993** [comment]
Fixed the rev-list selection
Please help review

**Comment by undera** [comment]
Looks good, thank you

**Pull request merged**